server/events/models/models.go

```diff
@@ -712,8 +714,22 @@ func (c CommandName) TitleString() string {
 	return strings.Title(strings.ReplaceAll(strings.ToLower(c.String()), "_", " "))
 }
 
+type PullContext struct {
+	PullNum     int
+	Repo        string
+	ProjectName string
+	Workspace   string
+}
+
+type JobContext struct {
+	PullContext
+	HeadCommit string
+}
```

Comment:
anyway we can have this in separate files? I hate this file, it's way too big and `models` doesn't make any sense. See the [style guide](https://docs.lyft.net/eng/atlantis/internal/team_practices/style_guide.html#organize-by-responsibility) for what i mean.

Reply:
I was thinking of refactoring all the log streaming logic into jobs package in server directory. I'll create a models.go in that package and move all the model structs used by log streaming into that file. Since it would be a big change that involves moving files and renaming imports, I think it's best if I do it in a follow up PR. What do you think?

Reply:
i still don't like the word `models` to encompass all our data models, let's name with purpose.  I would even be ok with just doing something as simple as keeping this in `project_command_output_handler` or some file that's related to where this is used.

Reply:
I've moved those structs to `project_command_output_handler`!